shuup/default_reports/reports/customer_sales.py

```diff
@@ -30,7 +30,7 @@ class CustomerSalesReport(OrderReportMixin, ShuupReportBase):
 
     def get_objects(self):
         return Contact.objects.filter(
-            customer_orders__in=super(CustomerSalesReport, self).get_objects()
+            customer_orders__in=super(CustomerSalesReport, self).get_objects()[:self.report_limit]
```

Comment:
Could you use `settings.DEFAULT_REPORTS_ITEM_LIMIT` instead `from django.conf import settings`?This way we do not need code changes to up the limit. Applies to all reports.